index.bs

```diff
@@ -12521,6 +12525,24 @@ values passed to \[[Call]]:
 </ol>
 
 
+<h4 id="preventextensions">Platform object \[[PreventExtensions]] method</h4>
+
+When the \[[PreventExtensions]] internal method of a [=platform object=] |O| is called,
+the following steps are taken:
+
+<ol class="algorithm">
+    1.  If |O| [=support indexed properties|supports indexed properties=] or
+        |O| [=support named properties|supports named properties=], 
+        return <emu-val>false</emu-val>.
+    1.  Return [=OrdinaryPreventExtensions=](|O|).
+</ol>
+
+Note: this turns the internal \[[PreventExtensions]] method
+of [=platform objects=] which [=support indexed properties|support indexed=] or
+[=support named properties|named properties=] into a noop,
+keeping them extensible, as failing to do so would make them non-functional.
```

Comment:
The reason is more that it would cause [[GetOwnProperty]] to violate [the invariants of internal methods](https://tc39.github.io/ecma262/#sec-invariants-of-the-essential-internal-methods) (last invariant in the GOP section), since in an object with named/indexed properties, GetOwnProperty returns new properties all the time (whenever the underlying data changes).